frontend/src/util/Utility.ts

```diff
@@ -0,0 +1,19 @@
+import SockJS from "sockjs-client";
```

Comment:
Could this maybe go in the `api` folder, since this [Mozilla](https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API) page refers to it as an API? Or maybe we could rename it to `sockets` or something, mainly cause to me this file seems more important than just a utility class.

Reply:
Sure, that sounds good -- moved it to the `api` folder in a `Socket.ts` file.

I also renamed `problem.ts` to `Problem.ts` to match -- we've used capitalized files in the frontend throughout this repository, I think that change keeps this uniform.